modules/guardian/types.go

```diff
@@ -2,37 +2,101 @@ package guardian
 
 import (
 	sdk "github.com/irisnet/irishub/types"
+	"github.com/pkg/errors"
+	"fmt"
+	"encoding/json"
 )
 
-type Profiler struct {
-	Name      string         `json:"name"`
-	Addr      sdk.AccAddress `json:"addr"`
-	AddedAddr sdk.AccAddress `json:"added_addr"`
+type Guardian struct {
+	Description  string         `json:"description"`
+	AccountType  AccountType    `json:"type"`
+	Address      sdk.AccAddress `json:"address"`
+	AddedAddress sdk.AccAddress `json:"added_address"`
 }
```

Comment:
Description string
AccountType AccountType
Address sdk.AccAddress     // this guardian's address
AddedBy sdk.AccAddress     // address that initiated the AddGuardian tx